### DISCUSSION TITLE
cuMemSetAccess entry point

### DISCUSSION THREAD

**Discussion by ElsayedMMohammed** [NVlabs/instant-ngp#1131] [Q&A]
When I try to execute testbe.exe on windows (.\build\testbed -- scene data\nerf\fox), I get the following error:
Entry Point Not Found, The Procedure entry point cuMemSetAccess Entry Point Could Not Be Located The Dynamic Link Library.

You know why?

Thanks

**Comment by dustindCWG** [comment]
I'm facing the same issue trying to execute instant-ngp `.\build\instant-ngp --scene data\nerf\fox`

**Comment by Tom94** [comment]
Which compiler version, CUDA version, and driver version are you running on?

**Reply by dustindCWG** [reply]
Hi @Tom94 - 

Compiler =  C/C++ Optimizing Compiler Version 19.34.31937
CUDA = Cuda compilation tools, release 12.0, V12.0.76, Build cuda_12.0.r12.0/compiler.31968024_0
Driver = 25.21.14.2525 for a Tesla K80

Please let me know if I can provide any more information

**Reply by Tom94** [reply]
That‘s good to know, thanks. I haven’t tested CUDA 12 yet and the error is new to me, so I suspect it might be connected. Do you have the ability to try an earlier 11.X version, say, 11.8?

I’ll also dig deeper after Christmas.

**Reply by dustindCWG** [reply]
A few changes did seem to help, but now I'm running into "GLFW error #65542: WGL: The driver does not appear to support OpenGL" when I try to fire up the fox demo.

Currently I have: 
- Windows Server 2019, running on an Azure VM (NC6) that I'm connecting to via Remote Desktop from my Mac
- Visual Studio 2019
- CUDA 11.8
- Nvidia Driver 474.14

Is running this on a VM and connecting to it via Remote Desktop something that would be supported?
